minari/dataset/minari_dataset.py

```diff
@@ -243,7 +199,9 @@ def set_seed(self, seed: int):
         """Set seed for random episode sampling generator."""
         self._generator = np.random.default_rng(seed)
 
-    def filter_episodes(self, condition: Callable[[h5py.Group], bool]) -> MinariDataset:
+    def filter_episodes(
+        self, condition: Callable[[EpisodeData], bool]
+    ) -> MinariDataset:
         """Filter the dataset episodes with a condition.
 
         The condition must be a callable with  a single argument, the episode HDF5 group.
```

Comment:
remove mention to HDF5, now should be EpisodeData

Reply:
done